test/plugin/scenarios/webflux-scenario/support-version.list

```diff
@@ -16,5 +16,10 @@
 
 # 2.0.0-2.1.0 are supported, but due to the status code bug(https://github.com/spring-projects/spring-framework/issues/21901)
 # we don’t test them
-
+2.1.1.RELEASE
+2.1.2.RELEASE
+2.1.3.RELEASE
+2.1.4.RELEASE
+2.1.5.RELEASE
+2.1.6.RELEASE
```

Comment:
Recently, we are reducing the unnecessary tests, you could run all these locally, but in the upstream CI, we recommend the latest mini version num, 2.1.6 for all 2.1.x.

Reply:
Need I modify it to 2.1.6 to 2.1.15 ? Just like this:
```txt
2.1.6.RELEASE
2.1.7.RELEASE
2.1.8.RELEASE
2.1.9.RELEASE
2.1.10.RELEASE
2.1.11.RELEASE
2.1.12.RELEASE
2.1.13.RELEASE
2.1.14.RELEASE
2.1.15.RELEASE
```

Reply:
I think just 2.1.15? Do they have big difference?

Reply:
The differences between 2.1.x versions are mainly bug fixes. It seems that the difference will not affect our test results.But the version has released to 2.3.x and these versions will include new feature additions . So I don't whether we need to add test for different version of 2.x ?
The specific differences can be seen in the following link：https://github.com/spring-projects/spring-boot/tags

Reply:
Basically, we recommend using the latest versions of 2.1.x, 2.2.x, and 2.3.x

Reply:
ok,I got it . I wiil modify it immediately.